### PR TITLE
IT-5088 | ProgressBar add badge + refactor range colors [breaking]

### PR DESCRIPTION
**Pull request by kvas-damian**
Breaking changes:
* `numberOfLayers, colorScheme and color` props are removed -> migrate to `color` attribute in `ranges` props

### CODE REVIEW

File: lib/js/components/ProgressBar/ProgressBar.vue [hunk 1]
```diff
@@ -27,21 +24,39 @@
 			</div>
 		</div>
 		<div
+			class="progressBar__barWrapper"
 			:class="{
-				progressBar__bar: true,
 				'-small': size === PROGRESS_BAR_SIZES.SMALL,
 				'-xsmall': size === PROGRESS_BAR_SIZES.XSMALL,
 				'-noRadius': radius === PROGRESS_BAR_RADII.NONE,
```
Comment (rogatty) [inline lib/js/components/ProgressBar/ProgressBar.vue:31]: It's no longer used.

Comment (kvas-damian) [reply lib/js/components/ProgressBar/ProgressBar.vue:31]:
```suggestion
```

File: lib/js/components/ProgressBar/ProgressBar.vue [hunk 2]
```diff
@@ -239,55 +247,79 @@ $progress-bar-layers: (
...
+		labelTextSize: {
 			type: String,
-			default: PROGRESS_BAR_SIZES.MEDIUM,
+			default: PROGRESS_BAR_LABEL_TEXT_SIZES.SMALL,
 			validator(size) {
-				return Object.values(PROGRESS_BAR_SIZES).includes(size);
+				return Object.values(PROGRESS_BAR_LABEL_TEXT_SIZES).includes(size);
 			},
 		},
 		ranges: {
```
Comment (rogatty) [inline lib/js/components/ProgressBar/ProgressBar.vue:325]: What do you think about adding a validator here? Maybe we should add some range limits, so https://github.com/bethinkpl/design-system/blob/d7b039c8615d047c040658a80faa8dfc263d915f/lib/js/components/ProgressBar/ProgressBar.vue#L44 doesn't blow up?

File: lib/js/components/ProgressBar/ProgressBar.stories.ts [hunk 3]
```diff
@@ -19,43 +19,37 @@ export default {
 const StoryTemplate: StoryFn<typeof ProgressBar> = (argTypes) => ({
 	components: { ProgressBar },
 	props: Object.keys(argTypes),
-	template: `<ProgressBar v-bind=$props></ProgressBar>`,
+	template: `
+			<ProgressBar v-bind=$props
+									 :badge-position="badgePosition === '' ? null : parseInt(badgePosition, 10)"></ProgressBar>`,
```
Comment (rogatty) [inline lib/js/components/ProgressBar/ProgressBar.stories.ts:24]: Weird formatting.

Comment (rogatty) [inline lib/js/components/ProgressBar/ProgressBar.stories.ts:24]: All stories apart from interactive one throw warnings:
```
Property or method "badgePosition" is not defined on the instance but referenced during render.
Invalid prop: custom validator check failed for prop "badgePosition".
```

File: lib/js/components/ProgressBar/ProgressBar.vue (hunk 2 above)
Comment (kvas-damian) [reply lib/js/components/ProgressBar/ProgressBar.vue:325]: [`01deed9` (#268)](https://github.com/bethinkpl/design-system/pull/268/commits/01deed93b0ecf013ea69b1a6db761ae6a39babe4)

File: lib/js/components/ProgressBar/ProgressBar.stories.ts (hunk 3 above)
Comment (kvas-damian) [reply lib/js/components/ProgressBar/ProgressBar.stories.ts]: [`150ef24` (#268)](https://github.com/bethinkpl/design-system/pull/268/commits/150ef24514980456a8aba36f60f75fbf35ac5cf9)

**Review by rogatty — APPROVED** [review]

**Pull request merged**